libsolidity/parsing/Parser.cpp

```diff
@@ -22,11 +22,11 @@
 
 #include <cctype>
 #include <vector>
-#include <liblangutil/SourceLocation.h>
 #include <libsolidity/parsing/Parser.h>
-#include <liblangutil/Scanner.h>
-#include <libsolidity/inlineasm/AsmParser.h>
+#include <libyul/AsmParser.h>
+#include <liblangutil/SourceLocation.h>
```

Comment:
Could be swapped with the one below ;)

Reply:
Which line exactly? the sourcelocation include-line with ErrorReporter? I'd say I don't think that's relevant unless we want to alphabetically sort grouped includes. :)

Reply:
Ok, I thought we aim to have the includes sorted alphabetically. But it's also fine like that :)